doc/progress.rst

```diff
@@ -8,6 +8,7 @@ Changelog
 
 0.10.1
 ~~~~~~
+* ADD #857: Adds task type ID to list_runs
```

Comment:
This is actually my mistake (as I didn't update this yet), but since we made a new release, you need to put this under a new header.